lib/surveyor/models/survey_section_methods.rb

```diff
@@ -7,9 +7,9 @@ module SurveySectionMethods
 
       included do
         # Associations
-        has_many :questions, -> { order( 'display_order, id ASC' ) }, :dependent => :destroy, :autosave => true
-        has_many :skip_logics, -> { order( 'execute_order, id ASC' ) }, :dependent => :destroy, :autosave => true, :inverse_of => :survey_section
-        belongs_to :survey
+        has_many :questions, -> { order( 'display_order, id ASC' ) }, dependent: :destroy, autosave: true
+        has_many :skip_logics, -> { order( 'execute_order, id ASC' ) }, dependent: :destroy, autosave: true, inverse_of: :survey_section
+        belongs_to :survey, required: false
```

Comment:
I think this is probably also one that should be required

Reply:
Again -- due to the nature of the parser this isn't always true. I'm happy to come back to this but my hope with this PR was mostly to get this back to parity w/ 4.2 logic (which was required: false by default, even if it doesn't make all that much sense in our usage).